int32 index
int32 sync
float32 ref_dist
float32 cur_dist
float32 send_vel
float32 steer_angle
